### DISCUSSION TITLE
如何在hexo的fluid主题中设置waline？

### DISCUSSION THREAD

**Discussion by ncwzdlsd** [walinejs/waline#1674] [Q&A] [answered]
如何在hexo的fluid主题中设置waline？谢谢

**Comment by lizheming** [comment, marked as answer]
https://www.zywvvd.com/notes/hexo/theme/fluid/fluid-waline/fluid-waline/ 

<img width="1045" alt="image" src="https://user-images.githubusercontent.com/424491/208298382-aafdac6c-f0e2-4c57-a10f-921e2bd51fb3.png">
